File: GFG/Array/SecondLargest.java
public class SecondLargest {
    public int getSecondLargest(int[] arr) {
        // code here
        int max,max2;
        max=arr[0];
        max2=-1;
        
        for(int n:arr){
            if(n>max){
                max2=max;
                max=n;
            }
            else if(n>max2 && n<max){
                max2=n;
            }
        }
        
        return max2;
    }

}

File: GFG/Array/Day1/SecondLargest.py
class Solution:
    def getSecondLargest(self, arr):
        # Code Here
        max1=arr[0]
        max2=-1
        
        for num in arr:
            if(num>max1):
                max2=max1
                max1=num
            elif(num>max2 and num<max1):
                max2=num
                
        return max2